tests/shell/ShellCheckBearTest.py

```diff
@@ -17,7 +17,27 @@
 echo $X
 """
 
+valid_with_ignore_rules_file = """
+#!/usr/bin/env sh
+echo "Path is $PATH"
+cd /home
+for dir in */
+do
+  cd "$dir"
+  convert index.png index.jpg
+  cd ..
+done
+"""
+
+ShellCheckBear1Test = verify_local_bear(
```

Comment:
Using `1` and `2` is not descriptive.

Keep this test named `ShellCheckBearTest`, as that avoids unnecessary modifications being added to your patch, and means `git blame` continues to show the authors of these tests.

btw, nice adding `valid_with_ignore_rules_file` to the invalid list of this test.